suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml

```diff
@@ -542,6 +528,64 @@ tests:
       module: sanity_rgw_multisite.py
       name: m buckets with n objects
       comments: bug-2213138, known failure
+# test the workaround to sync only from one active zone
+  - test:
+      abort-on-fail: true
```

Comment:
@viduship, could you remove the abort-on-fail flag, as it might block other tests from execution.

Reply:
sure, done.